prog/vnet/update_load_balancer.rb

```diff
@@ -18,7 +18,10 @@ def load_balancer
   end
 
   label def remove_load_balancer
-    vm.vm_host.sshable.cmd("sudo ip netns exec #{vm.inhost_name} nft --file -", stdin: generate_nat_rules(vm.ephemeral_net4.to_s, vm.nics.first.private_ipv4.network.to_s))
+    if vm
```

Comment:
In the next commit you already add this check to `before_run`. Do we need this extra branch and test here?

```ruby
  def before_run
    pop "VM is destroyed" unless vm
  end
```